discretisedfield/mesh.py

```diff
@@ -1505,3 +1544,144 @@ def axis_selector(self, *, widget="dropdown", description="axis"):
             description=description,
             disabled=False,
         )
+
+    def allclose(self, other, rtol=1e-05, atol=1e-08):
```

Comment:
We have to think about better defaults. The default `atol` is much bigger that the typical cell size (order 1e-9).